browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1305,6 +1307,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		checkbox.type = 'checkbox';
 		checkbox.id = data.id + '-input';
 		checkbox.tabIndex = '0';
+		checkbox.setAttribute('autocomplete', 'off');
```

Comment:
possibly not needed here (checkbox is not a text field)

Reply:
Made the update on this